console/react/src/overview/dashboard/throughputChart.test.js

```diff
@@ -23,7 +23,10 @@ import { service, login, TEST_PORT } from "../../serviceTest";
 import ThroughputChartData from "./throughputData";
 import ThroughputChart from "./throughputChart";
 
-it("renders the ThroughputChart component", () => {
+it("renders the ThroughputChart component", (done) => {
```

Comment:
The test is running assertions in a callback, so it needs to use the callback-friendly version of Jest test method, with `done()` callback. Otherwise, it is possible that Jest ends the test before the callback with the asserts has a chance to run. This is why the test was flaky before.